xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -42,7 +42,6 @@
 #include "settings/Settings.h"
 #include "FileItem.h"
 #include "guilib/LocalizeStrings.h"
-#include "utils/StringUtils.h"
```

Comment:
Good point. I am all for removing/minimising Util classes, with your approach it would be easier.